task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -0,0 +1,60 @@
+import time
+from threading import Thread
+
+from pyrsistent import m
+from six.moves.queue import Queue
+
+from task_processing.interfaces.task_executor import TaskExecutor
+
+
+class RetryingExecutor(TaskExecutor):
```

Comment:
I think the same argument could be made for any of the executors. I guess we're just providing a set of implementations for common patterns for running tasks. I think this is fine.